Cod3rsGrowth.Web/wwwroot/webapp/controller/Base.js

```diff
@@ -45,5 +45,13 @@ sap.ui.define([
         _vincularRota: function(Rota, Metodo){
             this._getRouter().getRoute(Rota).attachMatched(Metodo, this); 
         },
+
+        _pegarModelo: function(NomeDoModelo){
```

Comment:
nome de paramentro de funcao começa com minusculo e vc pode criar uma função assim
_modelo: function(nome, modelo){
if(modelo){
 return this.getView().setModel(modelo, nome)
}
return this.getView().getModel(nome);
}